src/pages/Cart/Cart.js

```diff
@@ -1,6 +1,89 @@
-import { useDocumentTitle } from "../../hooks/useDocumentTitle"
-
+import axios from "axios";
+import { CartPriceCard, EmptyCart, HorizontalCard } from "../../components";
+import { useCart } from "../../contexts";
+import { useDocumentTitle } from "../../hooks/useDocumentTitle";
+import { encodedToken } from "../../utilities/token";
+import "./Cart.css";
 export function Cart() {
-    useDocumentTitle("Cart - Watches World")
-    return(<div>This is Cart Page</div>)
-};
+  const { cartItems, setCartItems } = useCart();
+
+  const removeFromCartHandler = async (productId) => {
```

Comment:
add the API calling function in a separate file and outside a component

Reply:
okk will do in refactoring part